resources/rest-service/cloudify/migrations/versions/396303c07e35_5_2_to_5_3.py

```diff
@@ -23,14 +26,27 @@ def upgrade():
     _create_blueprints_labels_table()
     _modify_deployments_labels_table()
     _add_specialized_execution_fk()
+    _add_deployment_statues()
```

Comment:
This shoudl be _add_deployment_status, not statues

Reply:
typo, actually this should be `statuses` since we are adding more than status here, will update that to `_add_deployment_statuses`